### PR TITLE
Update sandman_zh_CN.ts

### PR DESCRIPTION
**Pull request by nkh0472**

### CODE REVIEW

File: SandboxiePlus/SandMan/sandman_zh_CN.ts [hunk 1]
```diff
@@ -5931,7 +5931,7 @@ Error: %1</source>
     <message>
         <location filename="Windows/SettingsWindow.cpp" line="1606"/>
         <source>Make Folder/File &amp;Forced</source>
-        <translation type="unfinished"></translation>
+        <translation>强制创建目录/文件</translation>
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/sandman_zh_CN.ts:5934]: 更改为“使得目录/文件强制进程化”

File: SandboxiePlus/SandMan/sandman_zh_CN.ts [hunk 2]
```diff
@@ -7828,12 +7828,12 @@ If you are a great patreaon supporter already, sandboxie can check online for an
     <message>
         <location filename="Forms/OptionsWindow.ui" line="1775"/>
         <source>Create a new sandboxed token instead of setting down default token</source>
-        <translation type="unfinished"></translation>
+        <translation>创建新的沙盒令牌，而不是设置默认令牌</translation>
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/sandman_zh_CN.ts:7831]: 创建新的沙盒令牌，而不是对默认令牌降权

File: SandboxiePlus/SandMan/sandman_zh_CN.ts [hunk 3]
```diff
@@ -9413,7 +9413,7 @@ Sandboxie 提供了针对这些情况的处理选项，可以在此页面进行
     <message>
         <location filename="Forms/SettingsWindow.ui" line="564"/>
         <source>Add ‘Make Folder/File Forced&apos; to the context menu</source>
-        <translation type="unfinished"></translation>
+        <translation>在上下文菜单添加“强制创建目录/文件”</translation>
```
Comment (love-code-yeyixiao) [inline SandboxiePlus/SandMan/sandman_zh_CN.ts:9416]: 在上下文菜单添加“使得目录/文件强制进程化”

File: SandboxiePlus/SandMan/sandman_zh_CN.ts (hunk 1 above)
Comment (nkh0472) [reply SandboxiePlus/SandMan/sandman_zh_CN.ts:5934]: need more supporting info

**Comment by love-code-yeyixiao** [comment]
I am the author of this feature,so I know the role of it.I think maybe none know more about this feature than me so far.

**Comment by love-code-yeyixiao** [comment]
When the user clicks on the function in this context menu and enters the name of the corresponding sandbox, Sandman adds the "Force Process" or "Force Directory" configuration to the corresponding sandbox configuration. If you don't want to use it directly, you can summarise it again yourself, but the description as originally submitted is definitely not correct.

**Comment by nkh0472** [comment]
> I am the author of this feature,so I know the role of it.I think maybe none know more about this feature than me so far.

I acknowledge your input as the author of this feature and your knowledge about it.

However, I still don't understand the exact purpose of this function from either the English version or your suggested Chinese translation, even after searching. Could you revise the English string or provide some clarification in the interface?

In the meantime, given your familiarity with both the feature and the Chinese language, I'll revert my translation for this function. Please go ahead and commit the appropriate translation.

**Comment by DavidXanatos** [comment]
I'll merge this and review the English version

**Pull request merged**